testing/gerbv_example/slots-milldrill/expected/milldrill.ngc

```diff
@@ -577,558 +577,303 @@ G1 X3.95183 Y-2.39173
-G1 Z-0.08889
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.11852
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.14815
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.17778
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.20741
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.23704
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.26667
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.29630
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.32593
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.35556
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.38519
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.41481
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.44444
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.47407
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.50370
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.53333
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.56296
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.59259
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.62222
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.65185
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.68148
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.71111
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.74074
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.77037
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.80000
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.82963
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.85926
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.88889
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.91852
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.94815
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-0.97778
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.00741
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.03704
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.06667
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.09630
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.12593
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.15556
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.18519
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.21481
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.24444
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.27407
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.30370
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.33333
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.36296
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.39259
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.42222
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.45185
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.48148
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.51111
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.54074
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.57037
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
-G1 Z-1.60000
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
+G1 Z0.02963
```

Comment:
Should this be a G0?  G0 is for fast moves that are not doing any drilling.  What do you think?

Reply:
I opted for the G1 because it's getting very close to the work piece and it's not autolevelled. So it could hit the surface if the PCB is uneven which I don't think is a good idea with G0.

Reply:
Good call.